tests/ops/op_math/test_controlled_ops.py

```diff
@@ -49,9 +49,61 @@
 X_broadcasted = np.array([X] * 3)
 
 
+# pylint: disable=too-many-public-methods
 class TestControlledQubitUnitary:
     """Tests specific to the ControlledQubitUnitary operation"""
 
+    def test_wires_is_none(self):
+        base_op = [[0, 1], [1, 0]]
+
+        with pytest.raises(TypeError, match="Must specify a set of wires"):
```

Comment:
To go with the previous suggested change
```suggestion
        with pytest.raises(ValueError, match="Must specify a set of wires"):
```

Reply:
Resolved in other conversation.